src/locale/locales/fr/messages.po

```diff
@@ -5250,31 +4878,31 @@ msgstr "Envoyer un fichier texte vers :"
 #: src/view/com/util/UserBanner.tsx:116
 #: src/view/com/util/UserBanner.tsx:119
 msgid "Upload from Camera"
-msgstr ""
+msgstr "Téléchargement à partir de l’appareil photo"
 
 #: src/view/com/util/UserAvatar.tsx:345
 #: src/view/com/util/UserBanner.tsx:133
 msgid "Upload from Files"
-msgstr ""
+msgstr "Téléchargement à partir de fichiers"
 
 #: src/view/com/util/UserAvatar.tsx:339
 #: src/view/com/util/UserAvatar.tsx:343
 #: src/view/com/util/UserBanner.tsx:127
 #: src/view/com/util/UserBanner.tsx:131
 msgid "Upload from Library"
-msgstr ""
+msgstr "Téléchargement à partir de la bibliothèque"
```

Comment:
Let's be fancy and use `photothèque` here to better mean "that library where there are pictures"!

```suggestion
msgstr "Envoyer à partir de la photothèque"
```